README.md

```diff
@@ -8,25 +8,35 @@ http://oss.oracle.com/licenses/upl.
 
 -----
 
-# Public Slack Channel Available
+# Oracle Coherence Operator
```

Comment:
Please drop the word Oracle.  After multiple discussions on naming, we have consensus on "Coherence Operator" and we want the nomenclature to be consistent everywhere.

Reply:
fixed